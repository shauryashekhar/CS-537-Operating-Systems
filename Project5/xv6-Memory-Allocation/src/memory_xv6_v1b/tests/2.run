cd v1b; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_2; cd ..
