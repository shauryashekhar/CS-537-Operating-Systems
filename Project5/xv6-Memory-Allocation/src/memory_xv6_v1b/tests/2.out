spawn make CPUS=1 -f Makefile.test qemu-nox
qemu-system-i386 -nographic -drive file=fs.img,index=1,media=disk,format=raw -drive file=xv6.img,index=0,media=disk,format=raw -smp 1 -m 512 
xv6...
cpu0: starting 0
sb: size 1000 nblocks 941 ninodes 200 nlog 30 logstart 2 inodestart 32 bmap start 58
init: starting sh
$ test_2
Frames: DFFF PIDs: 1
Frames: DFFD PIDs: -2
Frames: DFFB PIDs: -2
Frames: DFF9 PIDs: -2
Frames: DFF7 PIDs: -2
Frames: DFF5 PIDs: -2
Frames: DFF3 PIDs: -2
Frames: DFF1 PIDs: -2
Frames: DFEF PIDs: -2
Frames: DFED PIDs: -2
Frames: DFEB PIDs: -2
Frames: DFE9 PIDs: -2
Frames: DFE7 PIDs: -2
Frames: DFE5 PIDs: -2
Frames: DFE3 PIDs: -2
Frames: DFE1 PIDs: -2
Frames: DFDF PIDs: -2
Frames: DFDD PIDs: -2
Frames: DFDB PIDs: -2
Frames: DFD9 PIDs: -2
Frames: DFD7 PIDs: -2
Frames: DFD5 PIDs: -2
Frames: DFD3 PIDs: -2
Frames: DFD1 PIDs: -2
Frames: DFCF PIDs: -2
Frames: DFCD PIDs: -2
Frames: DFCB PIDs: -2
Frames: DFC9 PIDs: -2
Frames: DFC7 PIDs: -2
Frames: DFC5 PIDs: -2
Frames: DFC3 PIDs: -2
Frames: DFC1 PIDs: -2
Frames: DFBF PIDs: -2
Frames: DFBD PIDs: -2
Frames: DFBB PIDs: -2
Frames: DFB9 PIDs: -2
Frames: DFB7 PIDs: -2
Frames: DFB5 PIDs: -2
Frames: DFB3 PIDs: -2
Frames: DFB1 PIDs: -2
Frames: DFAF PIDs: -2
Frames: DFAD PIDs: -2
Frames: DFAB PIDs: -2
Frames: DFA9 PIDs: -2
Frames: DFA7 PIDs: -2
Frames: DFA5 PIDs: -2
Frames: DFA3 PIDs: -2
Frames: DFA1 PIDs: -2
Frames: DF9F PIDs: -2
Frames: DF9D PIDs: -2
Frames: DF9B PIDs: -2
Frames: DF99 PIDs: -2
Frames: DF97 PIDs: -2
Frames: DF95 PIDs: -2
Frames: DF93 PIDs: -2
Frames: DF91 PIDs: -2
Frames: DF8F PIDs: -2
Frames: DF8D PIDs: -2
Frames: DF8B PIDs: -2
Frames: DF89 PIDs: -2
Frames: DF87 PIDs: -2
Frames: DF85 PIDs: -2
Frames: DF83 PIDs: -2
Frames: DF81 PIDs: -2
Frames: DF7F PIDs: -2
Frames: DF7D PIDs: -2
Frames: DF7B PIDs: -2
Frames: DF79 PIDs: -2
Frames: DF77 PIDs: -2
Frames: DF75 PIDs: -2
Frames: DF73 PIDs: -2
Frames: DF71 PIDs: -2
Frames: DF6F PIDs: -2
Frames: DF6D PIDs: -2
Frames: DF6B PIDs: -2
Frames: DF69 PIDs: -2
Frames: DF67 PIDs: -2
Frames: DF65 PIDs: -2
Frames: DF63 PIDs: -2
Frames: DF61 PIDs: -2
Frames: DF5F PIDs: -2
Frames: DF5D PIDs: -2
Frames: DF5B PIDs: -2
Frames: DF59 PIDs: -2
Frames: DF57 PIDs: -2
Frames: DF55 PIDs: -2
Frames: DF53 PIDs: -2
Frames: DF51 PIDs: -2
Frames: DF4F PIDs: -2
Frames: DF4D PIDs: -2
Frames: DF4B PIDs: -2
Frames: DF49 PIDs: -2
Frames: DF47 PIDs: -2
Frames: DF45 PIDs: -2
Frames: DF43 PIDs: -2
Frames: DF41 PIDs: -2
Frames: DF3F PIDs: -2
Frames: DF3D PIDs: -2
Frames: DF3B PIDs: -2
Frames: DF39 PIDs: -2
Frames: DF37 PIDs: -2
Frames: DF35 PIDs: -2
Frames: DF33 PIDs: -2
Frames: DF31 PIDs: -2
Frames: DF2F PIDs: -2
Frames: DF2D PIDs: -2
Frames: DF2B PIDs: -2
Frames: DF29 PIDs: -2
Frames: DF27 PIDs: -2
Frames: DF25 PIDs: -2
Frames: DF23 PIDs: -2
Frames: DF21 PIDs: -2
Frames: DF1F PIDs: -2
Frames: DF1D PIDs: -2
Frames: DF1B PIDs: -2
Frames: DF19 PIDs: -2
Frames: DF17 PIDs: -2
Frames: DF15 PIDs: -2
Frames: DF13 PIDs: -2
Frames: DF11 PIDs: -2
Frames: DF0F PIDs: -2
Frames: DF0D PIDs: -2
Frames: DF0B PIDs: -2
Frames: DF09 PIDs: -2
Frames: DF07 PIDs: -2
Frames: DF05 PIDs: -2
Frames: DF03 PIDs: -2
Frames: DF01 PIDs: -2
Frames: DEFF PIDs: -2
Frames: DEFD PIDs: -2
$ QEMU: Terminated
